lib/onetime.rb

```diff
@@ -67,8 +67,9 @@ def load!(mode = nil, _base = Onetime::HOME)
       OT::ERRNO.freeze unless OT::ERRNO && OT::ERRNO.frozen?
       OT::Utils.fortunes ||= File.readlines(File.join(Onetime::HOME, 'etc', 'fortunes'))
       info "---  ONETIME #{OT.mode} v#{OT::VERSION}  -----------------------------------"
+      info "Sysinfo: #{@sysinfo.platform} (#{RUBY_VERSION})"
```

Comment:
Consider adding a log message to indicate the Ruby version being used. This can help with debugging and troubleshooting. [medium]